bin/protoc-go.sh

```diff
@@ -3,21 +3,31 @@
 set -eu
 
 gen() {
-
-    rm -rf controller/gen/common controller/gen/controller controller/gen/public controller/gen/config
-    mkdir -p controller/gen
-
     for f in $@; do
-        bin/protoc -I proto --go_out="plugins=grpc:$GOPATH/src" "$f"
+        bin/protoc -I proto --go_out=plugins=grpc,paths=source_relative:controller/gen "$f"
     done
-
-    git add controller/gen
```

Comment:
Nice. I also didn't like protoc messing with git :+1:

Reply:
@alpeb then you're not going to like line 33 😢

Reply:
oh haha, I missed that.